docs/sources/reference/cli/run.md

```diff
@@ -164,6 +164,12 @@ original configuration.
 Include `--config.extra-args` to pass additional command line flags from the original format to the converter.
 Refer to [alloy convert][] for more details on how `extra-args` work.
 
+## Automatic setting of GOMEMLIMIT
```

Comment:
@clayton-cornell I have no idea where to put this doc. I could put it in the environment variable section we have going since it relates closely to that. Maybe around the kubernetes sections since that is where it will be used 90% of the time.

Reply:
I would lean towards adding it to the Env Var topic and linking too the env var topic from wherever in the Kubernetes topics.

This would also push things to get the first draft of the env var topic published (and we can expand it as needed vs trying to capture additional vars).

Reply:
Now that the Env Var topic is live, should the GOMEMLIMIT info be shifted over?

Reply:
I havent forgotten this, on my todo list after I get the patch release out.